uPortal-security/uPortal-security-authn/src/main/java/org/apereo/portal/security/provider/SimpleSecurityContext.java

```diff
@@ -53,6 +53,12 @@ public synchronized void authenticate() throws PortalSecurityException {
         this.isauth = false;
         if (this.myPrincipal.UID != null && this.myOpaqueCredentials.credentialstring != null) {
 
+            // Logs if an attempt is made to log into a local account
+            StringBuilder msg = new StringBuilder();
+            msg.append("An attempt to log into the local login has occurred. ");
+            msg.append("user=" + this.myPrincipal.UID);
```

Comment:
Advantage of String buffer is to avoid string concatenation. Either move the "user=" text to the end of the previous string so that this becomes `msg.append(this.myPrincipal.UID);` or drop the whole StringBuffer var and just use `final String msg = "..." + this.myPrincipal.UID;`. The later only makes since since the message is only concatenating strings once.